tools/SetupFlow/DevHome.SetupFlow/ViewModels/PackageViewModel.cs

```diff
@@ -153,8 +167,16 @@ public Uri GetLearnMoreUri()
         return new Uri("https://github.com/microsoft/winget-pkgs");
     }
 
-    partial void OnIsSelectedChanged(bool value) => SelectionChanged?.Invoke(null, this);
-
+    partial void OnIsSelectedChanged(bool value) => SelectionChanged?.Invoke(null, this);
+
+    partial void OnSelectedVersionChanged(string value)
+    {
+        if (IsSelected && !IsSelectable())

```

Comment:
I don't understand what this is for.  Looks odd to have a method that changes `IsSelected` to false if the version is selected but isn't selectable.  What is this for?

Reply:
Added comment to elaborate on such use case. This is the expected behavior if the user selected an already installed version of the package, which would then unselects the package. Example:

![gh](https://github.com/microsoft/devhome/assets/104940545/4b2a4689-9cdc-4ec1-a779-a7fe3c9dfb3c)